operator/src/test/java/org/keycloak/operator/KeycloakDeploymentE2EIT.java

```diff
@@ -322,4 +322,16 @@ public void testInitialAdminUser() {
         }
     }
 
+    @Test
+    public void testCustomImage() {
```

Comment:
I'm not really happy in ignoring a test if a property is not set as it can be easily accidentally skipped.
An alternative would be to push a valid custom image to a public, personal Docker registry ( something like: `docker.io/andreatp/augmented-testing-keycloak`) and fallback to it, wdyt?

Reply:
It's pretty standard that tests are skipped in our testsuite by default, unless properly configured. Custom image in public repo might be problematic as we can't assure that the image matches the operator.